app/views/course/condition/_conditions.html.slim

```diff
@@ -0,0 +1,20 @@
+h2
+  span = t('course.condition.header')
+
+div.dropdown.pull-right
+  button.btn.btn-primary.dropdown-toggle#new-condition-dropdown-btn [data-toggle="dropdown"
+    aria-haspopup="true"]
+    span = t('course.condition.new.header')
+  ul.dropdown-menu.dropdown-menu-right aria-labelledby="new-condition-dropdown-btn"
+    - Course::Condition::ALL_CONDITIONS.map(&:downcase).each do |condition_name|
```

Comment:
Because now you have the full class name, map with:
```ruby
Course::Condition::ALL_CONDITIONS.map { |condition| condition.demodulize.downcase }.each
```